test_requirements.txt

```diff
@@ -5,11 +5,11 @@ coverage==5.3
 cryptography==2.5
 ddt==1.1.1
 djangorestframework==3.12.2
-edx-celeryutils
-edx-lint
+edx-celeryutils==1.1.0
+edx-lint==5.0.0
 edx-opaque-keys
-edx-submissions
-jsonfield==2.1.1
+edx-submissions==3.3.1
+jsonfield==3.1.0
```

Comment:
FYI, there is some work going on to get jsonfield to support django 3.2. See https://github.com/edx/upgrades/issues/42

Reply:
Earlier we were using jsonfiled==2.1.1 which was support django>=1.11 but now 3.3.1 which supports django>=2.2
https://github.com/rpkilby/jsonfield/blob/3.1.0/setup.py#L18
https://github.com/rpkilby/jsonfield/blob/2.1.1/setup.py#L15

Reply:
@pdpinch here she already mentioned that it supports django 3.2
https://github.com/edx/upgrades/issues/42#issuecomment-897206335